<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Feetech Servo GUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/icons/icon.jpg</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="3">
     <widget class="QPushButton" name="setIDBtn">
      <property name="text">
       <string>Set ID</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Set Pos</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="setposSlider">
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QSpinBox" name="writeValue">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Current Pos</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSlider" name="currentposSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>-1</number>
      </property>
      <property name="maximum">
       <number>1023</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QSpinBox" name="writeReg"/>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="readByteBtn">
      <property name="text">
       <string>Read Byte</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QSpinBox" name="setposSpin">
      <property name="maximum">
       <number>1023</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSpinBox" name="regSpinbox"/>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="enableTorque">
      <property name="text">
       <string>Torque Enabled</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="labelPos">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QPushButton" name="writeByteBtn">
      <property name="text">
       <string>Write Byte</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelReadByte">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="setIdEdit">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="currentID">
      <property name="minimum">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Current ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnDefParams">
      <property name="text">
       <string>Set default params</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
